src/index.js

```diff
@@ -18,22 +28,43 @@ function isGetRequest({ payload: { params } }) {
   return params.method === 'GET';
 }
 
+function isPostRequest({ payload: { params } }) {
+  return params.method === 'POST';
+}
+
+function isPutRequest({ payload: { params } }) {
+  return params.method === 'PUT';
+}
+
 export default function httpCache() {
-  const { get, set, check } = cache();
+  const { get, set, check, clear } = cache();
 
   return next => action => {
+
+    let url;
+
+    if (isRequestType(action)) {
+      url = action.payload.url;
+      if (isPostRequest(action)) {
```

Comment:
Can clean this up, leftover from when I was handling PUTs and POSTs differently.